package.json

```diff
@@ -1,81 +1,79 @@
 {
-	"name": "gpg-indicator",
-	"displayName": "GPG Indicator",
-	"description": "Status indicator and unlocker for GPG signing key",
-	"keywords": [
-		"pgp",
-		"gpg",
-		"commit",
-		"multi-root ready"
-	],
-	"author": {
-		"name": "Weida Hong",
-		"url": "https://blog.bitisle.net"
-	},
-	"version": "0.6.2",
-	"publisher": "wdhongtw",
-	"engines": {
-		"vscode": "^1.45.0"
-	},
-	"categories": [
-		"Other"
-	],
-	"icon": "images/icon-color.png",
-	"activationEvents": [
-		"workspaceContains:/.git",
-		"onCommand:gpgIndicator.unlockCurrentKey"
```

Comment:
One activation event is dropped, is this change intended?

Reply:
Yes, it's the suggestion from VSCode:

![image](https://user-images.githubusercontent.com/9762652/222437903-34964da4-43c0-4585-a388-536da32339cf.png)

Reply:
I recheck the manual, [the auto generation for `activationEvents` is only available from vscode 1.74](https://code.visualstudio.com/api/references/activation-events#onCommand), I will add it back **in the next pr**, sorry for that.

Reply:
OK :D